### ISSUE TITLE
Keyboard Focus not always visible nor consistent on Clean

### ISSUE THREAD

**Issue by StephDriver** [BirkbeckCTP/janeway#4872]
Focus is not always visible on Clean theme. 

See in particular, home page, focus is in different colours for different places, and almost impossible to see it on the carousel buttons.

<img width="1408" height="152" alt="Image" src="https://github.com/user-attachments/assets/68ef1e5a-ae76-445e-ab44-65a5387a0654" />

<img width="1016" height="333" alt="Image" src="https://github.com/user-attachments/assets/1f78b32a-9910-4c98-908f-7ce7f94ba4d1" />

<img width="302" height="129" alt="Image" src="https://github.com/user-attachments/assets/e111f351-ae91-4462-813f-0d176f572bc1" />

fine to have change for dark/light backgrounds, but on the same background this should be consistent behaviour.
